iamchrissmith/air_bnb_clone
# User view all reservation

#### What does this PR do?
Adds Past and Declined reservations to the user dashboard

#### Migrations
NO

#### Where should the reviewer start?
`views/dashboard/index.html.erb`

#### How should this be manually tested?
1. Create a reservation
2. in rails console change its status to declined or finished
3. log in as a user
4. check dashboard under appropriate tab

#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
  - Do Environment Variables need to be set?
  - Any other deploy steps?
